spec/mysql2/statement_spec.rb

```diff
@@ -1,6 +1,6 @@
 require './spec/spec_helper'
 
-RSpec.describe Mysql2::Statement do
+RSpec.describe Mysql2::Statement do # rubocop:disable Metrics/BlockLength
```

Comment:
I would guess that if not for the tests, this length could be reduced dramatically:
https://github.com/brianmario/mysql2/blob/03ac203ff077591171e54f65190071a3ef84525a/.rubocop_todo.yml#L37-L38

Reply:
IMO that cop is just plain bad.

Reply:
All `Metrics` are.